packages/fxa-settings/src/components/ThirdPartyAuth/en.ftl

```diff
@@ -0,0 +1,10 @@
+## ThirdPartyAuth component
+## This is a component that is used to display a list of third party providers (Apple, Google, etc.)
+
+# This appears when a user has the option to authenticate via third party accounts in addition to their Firefox account. 
+# Firefox account login appears on top, and third party options appear on bottom. 
+# This string appears as a separation between the two, in the following order: "Enter your password" "Or"(this string) "Continue with Google"(continue-with-google-button) / "Continue with Apple"(continue-with-apple-button)
+options-or = Or
```

Comment:
```suggestion
third-party-auth-options-or = Or
```
The comment and string look good. I'd recommend tweaking the ID here to be a bit more specific to avoid issues with potential conflicts or unintentional re-use of the string in other contexts.